### ISSUE TITLE
only some ES classes in a project get transpiled

### ISSUE THREAD

**Issue by nlundquist** [stealjs/steal#1505]
__Reproducible:__ Always

__Description:__
When using native classes from an NPM dependency (e.g CanJS 6) they're not transpiled even though app modules _are_ transpiled. This is an issue when the app modules include classes that extend the classes from an NPM dependency, e.g: 
```
import StacheElement from 'can-stache-element';
class TodoCreate extends StacheElement {
    ...
}
```
This will result in an error along the lines of:
`Uncaught TypeError: Class constructor Child cannot be invoked without 'new'`

This is because the `StacheElement` class is _not_ transpiled, but the `TodoCreate` class _is_ transpiled and transpiled classes cannot extend native classes since transpiled classes try to call their base class like a function rather than invoking with `new`.

It's unclear what the correct solution to this problem is. Depending on the supported browsers for your app one solution may be to prevent class transpilation inside app modules. The other option would be to apply transpilations to NPM dependencies so all classes are transpiled.  Both have downsides. Stopping transpilation limits browser support, increasing transpilation increases bundle size and increases dev mode transpilation delays.

**Comment by matthewp** [comment]
I think the bug here is the way ES classes are transpiled to constructor functions. They are not compatible with ES base classes.

**Comment by matthewp** [comment]
Otherwise #1276 is needed.

**Comment by justinbmeyer** [comment]
Could we just avoid classes in our code?

**Comment by matthewp** [comment]
That wouldn't solve this steal issue, no.

**Comment by frank-dspeed** [comment]
i have a way to handle that but i need some extra info is Steal's output formart still Systemjs loader compatible or lets say SystemJS compatible when systemjs gets loaded as Steal?

**Comment by matthewp** [comment]
@frank-dspeed Sorry, I don't understand your question. can you rephrase?

**Comment by frank-dspeed** [comment]
# Conclusion 
as maybe some people know i am working on stuff like typescript rollup vscode.

- Class should not get automaticly transpiled. Or Needs to get Always Transpiled and WebComponents Polyfill loaded

As WebComponents Pollyfill is big and hugh It is more clever to transpil WebComponents into a other Formart like canjs-view-parser :D

The CanJS way is the most efficent and compatible that still exists. It is a WebComponents Pollyfill it self,
